src/chaosinventory/settings.py

```diff
@@ -28,16 +30,25 @@
 
 SECRET_KEY = config.get('django', 'secret', fallback=None)
 
+if SECRET_KEY is None:
+    logging.warn("No custom secret configured. New random secret will be generated!")
+    # random secret will be generated. This key may be persisted in the next section
+    SECRET_KEY = secrets.token_urlsafe()
+
 if not len(config.sections()):
```

Comment:
In this case, we should kill the process and throw an error, I'd suggest?
For me that would be a proper reaction for the "config file without secret" case, because this will be probably created by accident.